configure.ac

```diff
@@ -778,7 +778,7 @@ AS_IF([test x$with_lustre != xno],[
 AC_ARG_ENABLE([libfab],
         [AS_HELP_STRING([--enable-libfab],
                         [Use libfabric transport])],
-        [],[enable_libfab=no]
```

Comment:
I don't understand this change.
```
— Macro: AC_ARG_ENABLE (feature, help-string, [action-if-given], [action-if-not-given])

```
According to manual, the last one is "action-if-not-given".

Reply:
This is done temporarily to enable libfab by default on the jenkins run